GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifiedAccountDetailResult.m

```diff
@@ -85,16 +87,15 @@ - (void)updateVerifiedDetailsWithTokenResponse:(nullable OIDTokenResponse *)toke
   }
 }
 
-- (nullable NSString *)accessTokenString {
-  return self.verifiedAuthState.lastTokenResponse.accessToken;
-}
-
-- (nullable NSString *)refreshTokenString {
-  return self.verifiedAuthState.refreshToken;
+- (nullable GIDToken *)accessToken {
+  return [[GIDToken alloc]
+          initWithTokenString:self.verifiedAuthState.lastTokenResponse.accessToken
+               expirationDate:self.verifiedAuthState.lastTokenResponse.accessTokenExpirationDate];
 }
 
-- (nullable NSDate *)expirationDate {
-  return self.verifiedAuthState.lastTokenResponse.accessTokenExpirationDate;
+- (nullable GIDToken *)refreshToken {
+  return [[GIDToken alloc] initWithTokenString:self.verifiedAuthState.refreshToken
```

Comment:
Same here. I don't think we want to create a new token each time this getter is called.

Reply:
Same changes added above. Created upon instantiation.